src/main/MQ2Inlines.h

```diff
@@ -249,7 +249,7 @@ inline int GetFocusRangeModifier(const EQ_Spell* pSpell, ItemPtr& pItemOut)
 
 inline bool HasSkill(int nSkill)
 {
-	return pLocalPC ? pLocalPC->HasSkill(nSkill) : 0;
+	return pLocalPC ? pLocalPC->HasSkill(nSkill) : false;
```

Comment:
this can be written without the ternary `return pLocalPC && pLocalPC->HasSkill(nSkill)`

Reply:
I appreciate you pointing that out, sometimes i have a bad habit of not stepping back for the bigger "what are we trying to do" and instead see a small part of it

Reply:
what _are_ you trying to do? This isn't a functional change

Reply:
it was returning a  "0" for a bool function, so i was trying to clean it up to return false instead of 0